### PR TITLE
TF models

### PR DESCRIPTION
**Pull request by aditipurandare**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,3 +6,13 @@ A project to collect mainstream TF models with emphasis on tflite models which c
 | Model |Description |Publications|Variants|
 |--|--|--|--|
 | [mobilenet-v1](./mobilenet-v1) | ? | ? | [1.0_224_quant2](./mobilenet-v1/mobilenet_v1_1.0_224_quant.tflite), [224](./mobilenet-v1) |
+| [mobilenet-v2](https://www.tensorflow.org/lite/guide/hosted_models) | Image Classification | ? | [1.0_224_quant2](https://storage.googleapis.com/download.tensorflow.org/models/tflite_11_05_08/mobilenet_v2_1.0_224_quant.tgz) |
+| [inception-v4](https://www.tensorflow.org/lite/guide/hosted_models) | Image Classification | ? | [299_quant](https://storage.googleapis.com/download.tensorflow.org/models/inception_v4_299_quant_20181026.tgz) |
+| [Mnas](https://www.tensorflow.org/lite/guide/hosted_models) | Image Classification | ? | [1.3_224](https://storage.googleapis.com/download.tensorflow.org/models/inception_v4_299_quant_20181026.tgz) |
```
Comment (agostini01) [inline README.md]: It appears that Mnas network link points to Inception network.
Would you modify it?

**Review by agostini01 — CHANGES_REQUESTED** [review]
Great work! Almost ready to merge. Just need to fix the Mnas link.

File: README.md (hunk 1 above)
Comment (aditipurandare) [reply README.md]: Sorry about that, I fixed the link

**Comment by agostini01** [comment]
Great work!

**Comment by agostini01** [comment]
resolve #2 
resolve #1

**Comment by agostini01** [comment]
fix #2

**Pull request merged**